src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -3682,10 +3688,7 @@ private bool ScanXmlAttributeTextToken(ref TokenInfo info)
 
                 case '\r':
                 case '\n':
-                    this.ScanEndOfLine();
-                    info.StringValue = info.Text = TextWindow.GetText(false);
-                    info.Kind = SyntaxKind.XmlTextLiteralNewLineToken;
-                    this.MutateLocation(XmlDocCommentLocation.Exterior);
+                    ScanXmlTextLiteralNewLineToken(ref info);
```

Comment:
Doesn't look like the return value is used anywhere.  Think we should make it a void method.

Reply:
Agred.  Fixed.